doubly_linked_list/doubly_linked_list.py

```diff
@@ -74,7 +109,23 @@ def move_to_end(self, node):
     return self.head.value
 
   def delete(self, node):
-    pass
+    if node.next is None and node.prev is None and self.head is node and self.tail is node:
+      self.head = None
+      self.tail = None
+      return
+    #This is the head
+    if node.prev is None and node.next is not None:
+      self.head = node.next
+
```

Comment:
Don't forget to now set self.head.prev to None.  Would also be good to set node.next = None , in case somewhere else in the code still has a reference to node.